### PR TITLE
Update some work

### PR DESCRIPTION
**Pull request by ywu94**

### CODE REVIEW

File: Script/input_generate.py [hunk 1]
```diff
@@ -0,0 +1,315 @@
...
+
+
+
+
+
+
+
+
+
+
```
Comment (OwenHua666) [inline Script/input_generate.py]: Unnecessary lines.

File: Script/multi_seq_lstm_classifier.py [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+		self.classification_layer = MLP_Classification_Layer(self.mlp_inp_size, out_size, dropout=mlp_dropout)
+		
+	def forward(self, *args):
+		assert len(args)==self.n_extraction+1
+		
+		extract_buffer = [getattr(self, 'extraction_layer_{}.format(index)')(inp_embed, args[-1]) for index, inp_embed in enumerate(args[:-1])]
+		out = torch.cat(extract_buffer, 1)
+		out = self.classification_layer(out)
+		
+		return out
```
Comment (OwenHua666) [inline Script/multi_seq_lstm_classifier.py:99]: EOF

File: Script/sample_training_script_multi_seq_lstm_classifier.py [hunk 3]
```diff
@@ -0,0 +1,211 @@
...
+			logger.info('=========================')
+
+		# Train model
+		model.train()
+		train_running_loss, train_n_batch = 0, 0
+
+		for index, (label_artifact_path, seq_inp_target, seq_inp_path) in enumerate(train_inp_tuple, start=1):
+			train_loader = train_data_loader(label_artifact_path, seq_inp_target, seq_inp_path, w2v_registry)
+			train_iterator = iter(train_loader)
+			while True:
```
Comment (OwenHua666) [inline Script/sample_training_script_multi_seq_lstm_classifier.py:103]: Why `while True`?

**Review by OwenHua666 — COMMENTED** [review]
The training script is not working for me. No error message. Just show killed.

File: Script/sample_training_script_multi_seq_lstm_classifier.py (hunk 3 above)
Comment (ywu94) [reply Script/sample_training_script_multi_seq_lstm_classifier.py:103]: Cause it's using an iterator which will throw StopIteration exception when reaching the end

File: Script/input_generate.py (hunk 1 above)
Comment (ywu94) [reply Script/input_generate.py]: Removed

File: Script/multi_seq_lstm_classifier.py (hunk 2 above)
Comment (ywu94) [reply Script/multi_seq_lstm_classifier.py:99]: ? What do you mean

Comment (OwenHua666) [reply Script/multi_seq_lstm_classifier.py:99]: Nothing important. End of file line

**Pull request merged**